website/docs/overview/query/analytical-query.md

```diff
@@ -175,10 +190,18 @@ Valid functions are the same as the ones listed in [Valid Aggregate Functions](#
    [HOST]/api/db/query
 ```
 
+</TabItem>
+
+<TabItem value="graphql">
+
 ```graphql
 Not supported
```

Comment:
Not sure, but should we remove the instances where a lang isn't supported or leave this message. 
I think if we leave the option of this format, we should have a more verbose message so that people understand why it isnt supported. Might be a bit of a large task, so may need to push this to a later date/release though.

Reply:
Per our conversation. We'll keep them for now, but I will touch base with Brian and Andrew as to why they are not supported and we can possibly indicate why in the docs.